dashboard/v1.1/src/pages/Monitoring/Monitoring.tsx

```diff
@@ -42,11 +42,16 @@ const ServicesState: FC<{}> = () => {
       });
   };
   return (
-    <Switch
-      checked={isActive}
-      color="primary"
-      onChange={() => updateServicesState()}
-    />
+    <Tooltip
+      title={isActive ? 'Stop Monitoring' : 'Start Monitoring'}
+      aria-label={isActive ? 'Stop Monitoring' : 'Start Monitoring'}
+    >
```

Comment:
`(isActive ? 'Stop' : 'Start') + ' Monitoring'`

Or something similar should work.